deltachat-ffi/deltachat.h

```diff
@@ -371,6 +371,8 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    generate RSA 2048 keypair
  *                    DC_KEY_GEN_ED25519 (2)=
  *                    generate Ed25519 keypair
+ * - `max_attach_size` = Maximum size of a sent file
```

Comment:
we should 

- either make clear, that `max_attach_size` must already be subtracted by the typical overhead as base64 encoding and so on.

- or, we say that the user may specify the raw size and we will substract the typical overhead or our own (minus 1mb minus 25%). 

as max_attach_size, as a config-value, will be configureabe also through the provider-db, maybe the second variant  is better - as this is typically also the value written down on some provider pages etc.

Reply:
1. The setting is applied directly to file size,
2. Regular user is not aware of base64 encoding, so it would be misleading for the user, that let's say restriction is 25 mb, but 20mb file is rejected, because of the size.